core/src/main/resources/hudson/views/BuildButtonColumn/column_pl.properties

```diff
@@ -1,25 +0,0 @@
-# The MIT License
-#
-# Copyright (c) 2004-2017, Sun Microsystems, Inc., Damian Szczepanik
-#
-# Permission is hereby granted, free of charge, to any person obtaining a copy
-# of this software and associated documentation files (the "Software"), to deal
-# in the Software without restriction, including without limitation the rights
-# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
-# copies of the Software, and to permit persons to whom the Software is
-# furnished to do so, subject to the following conditions:
-#
-# The above copyright notice and this permission notice shall be included in
-# all copies or substantial portions of the Software.
-#
-# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
-# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
-# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
-# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
-# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
-# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
-# THE SOFTWARE.
-
-Build_scheduled=Zadanie zosta\u0142o zaplanowane
-Schedule_a_build=Dodaj zadanie {0} do kolejki
-Schedule_a_build_with_parameters=Uruchom z parametrami dla {0}
```

Comment:
@damianszczepanik There's a discussion currently going on in #2772 about the impact of that change on localizations. I'd like to know, would something like this, i.e. no effort done on retaining existing translations, annoy or upset you? Or would you consider it just part of ongoing development?